src/Libraries/RevitNodesUI/RevitTypes.cs

```diff
@@ -129,4 +129,39 @@ public class VerticalAlignment : CustomGenericEnumerationDropDown
         public VerticalAlignment() : base("Vertical Alignment", typeof(Autodesk.Revit.DB.VerticalAlignmentStyle)) { }
     }
 
+    [NodeName("Fill Patterns")]
```

Comment:
I think they are from some codes added for master branch, and during cherry-pick they're just added into the 123 branch, should I delete them?